core/lib/scmi-server/sub.mk

```diff
@@ -1,20 +1,20 @@
 # SCMI server library is built from SCP-firmware source tree.
 # The firmware is made of a framework, a product and modules.
 # Only modules used by firmware must be built, as stated by
-# CFG_SCPFW_MOD_* swtches. SCP-firmware needs a C source and
+# CFG_SCPFW_MOD_* switches. SCP-firmware needs a C source and
 # a header file to be generated to describe embedded modules.
 # This is done through cmake configuration of the package.
 # The toolchain build directive must also match the list of
 # embedded modules.
 
 scpfw-path = $(CFG_SCP_FIRMWARE)
-scpfw-product = $(CFG_SCMI_SCPFW_PRODUCT)
+scpfw-product = optee/$(CFG_SCMI_SCPFW_PRODUCT)
 scpfw-out-path := $(out-dir)/$(libdir)
 
 # This script was validated against SCP-firmware 2.11.0 development branch,
 # from commit f1d894921d76 ("product/optee-fvp: Add new OPTEE FVP product").
```

Comment:
Agreed. If I have a manifest PR I can test and merge both at the same time so that CI doesn't break.

Reply:
So to be clear the steps are:
1. SCP-firmware needs to be updated first (add `product/optee/sub.mk`)
2. Then add a temporary commit to this PR updating `.github/workflows/ci.yml` so that we can make sure CI passes (function `download_scp_firmware()`: checkout the proper SHA1).
3. This PR will receive review tags
4. Create a manifest PR
5. I will merge both PRs (excluding the `ci.yml` hack of course)